src/App.js

```diff
@@ -37,8 +37,25 @@ class App extends Component {
       .catch((err) => this.setState({ error: err.message }));
   }
 
+  getSingleMovie(id) {
+    this.setState({ loading: true });
+    let fetchUrl =
+      "https://rancid-tomatillos.herokuapp.com/api/v2/movies/" + id;
+    fetch(fetchUrl)
```

Comment:
Now that we have this working we might want to refactor some of this to see if we can make it a bit simpler for instance even though it was my cludgy workaround I'm not sure we need this fetchUrl step.